### DISCUSSION TITLE
Server options iceTransportPolicy, maxPacketLifeTime, and maxRetransmits

### DISCUSSION THREAD

**Discussion by paullouisageneau** [geckosio/geckos.io#123] [Ideas]
Hello, author of libdatachannel here. First, this is a cool project, thanks for using libdatachannel under the hood!

The Readme states:

> `iceTransportPolicy`, `maxPacketLifeTime`, and `maxRetransmits` have been removed from the ServerOptions because they are not supported by node-datachannel.

For your information, `iceTransportPolicy` has been introduced in libdatachannel v0.14, and  `iceTransportPolicy`, `maxPacketLifeTime`, and `maxRetransmits` are now available in node-datachannel v0.1.7, so you could re-add these options if you want.

**Comment by yandeu** [comment]
That is very cool. Thanks a lot! 😃

**Reply by yandeu** [reply]
Done (https://github.com/geckosio/geckos.io/commit/8a1ed8e0a0630ecbb12265928adfd56226ac2acb)

**Reply by paullouisageneau** [reply]
@yandeu Great :+1:
